test/test.ts

```diff
@@ -3,19 +3,21 @@
  */
 /// <reference types="@google/local-home-sdk" />
 import test from 'ava';
+import cbor from 'cbor';
```

Comment:
maybe we could use something simpler (and built in) for the test?

Reply:
I added this back because it works with the [sample](https://github.com/actions-on-google/smart-home-local).  I guess in the long run I should keep all the tests in this file more generic, but I think that falls into issue #1